### PR TITLE
[R3] Ask your questions

### PR DESCRIPTION
**Pull request by g-stamatis**
https://taskman.eionet.europa.eu/issues/147638

### CODE REVIEW

File: theme/themes/eea/elements/button.overrides [hunk 1]
```diff
@@ -59,6 +51,19 @@
     background-color: @secondaryButtonColor;
     color: @secondaryButtonColorHover;
   }
+
+  &.icon.button i {
+    font-size: 1.5rem;
```
Comment (ichim-david) [inline theme/themes/eea/elements/button.overrides:56]: Add this value as a variable

File: theme/themes/eea/elements/button.overrides [hunk 2]
```diff
@@ -59,6 +51,19 @@
...
+    opacity: 1;
+  }
+}
+
+.ui.button.secondary[class*='right labeled'].icon.button {
+  padding-right: 3em !important; // to apply to theme button paddings - button.less:856
```
Comment (ichim-david) [inline theme/themes/eea/elements/button.overrides:62]: Add this value as a variable

File: theme/themes/eea/elements/button.overrides [hunk 3]
```diff
@@ -59,6 +51,19 @@
...
+}
+
+.ui.button.secondary[class*='left labeled'].icon.button {
+  padding-left: 3em !important; // to apply to theme button paddings - button.less:856
```
Comment (ichim-david) [inline theme/themes/eea/elements/button.overrides:66]: Add this value as a variable

File: theme/themes/eea/elements/button.overrides [hunk 4]
```diff
@@ -59,6 +51,19 @@
     background-color: @secondaryButtonColor;
     color: @secondaryButtonColorHover;
   }
+
+  &.icon.button i {
```
Comment (ichim-david) [inline theme/themes/eea/elements/button.overrides:55]: any button should have the ability to add an icon not just the secondary, move this rule to the .ui.button basic style

**Review by ichim-david — CHANGES_REQUESTED** [review]
@g-stamatis  I had to make a commit at line 90 to add .ui instead of ui.
You need to better check your selectors to ensure you don't have these kinds of
mistakes

**Comment by g-stamatis** [comment]
[R3] Ask your questions
https://taskman.eionet.europa.eu/issues/147638

**Comment by g-stamatis** [comment]
Included in buttons.

**Pull request closed without merging**